identity/background.js

```diff
@@ -0,0 +1,233 @@
+
+let providers = {
+  facebook: {
+    clientId: '1094807753900920',
+    clientSecret: '3d4af261312493c4397e8003c9139500',
```

Comment:
Is this ok to be in the repo? Maybe a comment about what developers should do if they were copying and pasting?